src/Squader.Api/Areas/Advertisements/Controllers/AdvertisementsController.cs

```diff
@@ -1,22 +1,26 @@
 ﻿using System;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
 using Squader.Cqrs;
 using Squader.DomainModel.Advertisements.Commands;
 
 namespace Squader.Api.Areas.Advertisements.Controllers
 {
     public class AdvertisementsController : BaseApiController
     {
-        public AdvertisementsController(ICommandBus commandBus, IQueryBus queryBus) : base(commandBus, queryBus)
+        
+        public AdvertisementsController(ICommandBus commandBus, IQueryBus queryBus, ILogger<AdvertisementsController> logger) : base(commandBus, queryBus, logger)
         {
+            
         }
         
         [HttpGet("create")] //should http post, but get is easier for manual tests
         public async Task<IActionResult> CreateNewAdvertisementAsync()
         {
             var command = new CreateNewAdvertisementCommand("test", "test");
             await this.commandBus.ExecuteAsync(command);
+            logger.LogInformation("Logger works !");
```

Comment:
W wersji jaka ma iść na mastera powinno nie być kodu do sprawdzenia czy to działa 
Powinien być tylko kod pasujący do zadania